core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1598,6 +1598,17 @@ public String toString() {
       .setScope(Scope.SERVER)
       .setDisplayType(DisplayType.CREDENTIALS)
       .build();
+  public static final PropertyKey ABFS_MSI_ENDPOINT = stringBuilder(Name.ABFS_MSI_ENDPOINT)
+      .setDescription("MSI endpoint")
+      .setDefaultValue("http://169.254.169.254/metadata/identity/oauth2/token")
```

Comment:
We should not set an explicit IP address in the default value

Reply:
already change it